api_server/charity/routers.py

```diff
@@ -101,9 +104,54 @@ async def delete_charity(org_id: UUID, charity_service: CharityService = Depends
         charity_service: dependency as business logic instance.
 
     Returns:
-    ResponseBaseSchema object with CharityOutputSchema object as response data.
+        ResponseBaseSchema object with CharityOutputSchema object as response data.
     """
     await charity_service.delete_organisation(org_id)
     return ResponseBaseSchema(status_code=HTTP_200_OK,
                               data={"detail": "Organisation has been deleted successfully."},
                               errors=[])
+
+
+@charities_router.post("/{organisation_id}/managers/add", response_model=ResponseBaseSchema)
```

Comment:
That ```/add``` at the end is absolutely unnecessary and not restful.
Some good read: https://restfulapi.net/resource-naming/